packages/nextjs/test/integration/src/pages/someNamedComponent.tsx

```diff
@@ -0,0 +1,6 @@
+export const MyNamedPage = () => (
+  <h1>
+    This page exists to test the compatibility of our auto-wrapper with the option of having the `pages` directory
+    inside a `src` directory (https://nextjs.org/docs/advanced-features/src-directory)
```

Comment:
So the idea is that this page should not get wrapped, correct?

Reply:
Correct

Reply:
How do you handle this when using Redux Toolkit for example?